meson.build

```diff
@@ -36,12 +36,26 @@ if get_option('local') and get_option('default_library') == 'shared'
   rpath_exe = '$ORIGIN/../' + get_option('libdir')
 endif
 
+userconf = configuration_data()
+userconf.set10('HAVE__GETPID', false)
+have_getpid = cc.has_function('getpid', prefix: '#include <unistd.h>')
+userconf.set10('HAVE_GETPID', have_getpid)
+if not have_getpid
```

Comment:
is there any reason to make `HAVE__GETPID` defined only if `HAVE_GETPID` is not? I think it should work even without the condition.

Reply:
only reasons are better performance and that the logic makes more sense (i.e. `_getpid()` is the backup for `getpid()` so it's not needed if `getpid()` is available) but I'm fine with checking for `_getpid()` always 6a12026